### PR TITLE
Improve documentations for 4.4.0 release

### PR DESCRIPTION
**Pull request by AnuGayan**
## Purpose
This PR will add the following changes.
- Revamp the Getting Started guide
- Remove Streaming Integrator documentation
- Remove Micro Integrator documentation

Quick Start Guide

Screenshot 2024-10-30 at 3 30 20 PM
Screenshot 2024-10-30 at 3 30 33 PM

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -2051,636 +1086,19 @@ plugins:
...
-              'tutorials/integration-tutorials/using-the-gmail-connector.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/using-the-gmail-connector'
-
+              'design/prototype-api/create-a-mock-api-with-an-inline-script.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/create-a-prototype-api/'
+              'design/prototype-api/deploy-and-test-mock-apis.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/create-a-prototype-api/'
+              'design/prototype-api/create-a-prototype-api.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/overview/'
+              'deploy-and-publish/deploy-on-gateway/api-gateway/message-tracing.md': 'https://apim.docs.wso2.com/en/4.2.0/get-started/about-this-release/#removed-features-and-functionalities'
+              'develop/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping.md': 'https://apim.docs.wso2.com/en/4.2.0/reference/customize-product/customizations/customizing-the-developer-portal/customize-api-listing/api-category-based-grouping/'
```
Comment (ashera96) [inline en/mkdocs.yml:1101]: The version within the path should be 4.4.0. Shall we revert the change done to these lines?

**Comment by isamauny-wso2** [comment]
If possible I would also add Podman command, since Docker might be restricted at some customers.

**Review by ashera96 — APPROVED** [review]
